net-misc/gallery-dl/gallery-dl-1.17.5.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=(python3_{7..9})
+PYTHON_COMPAT=(python3_{8,9})
```

Comment:
This isn't worth its own commit, 7 does nothing (can't be selected anymore), and eventually will be a `*/*` commit to remove all dead compat in the tree at once.

Wouldn't say if you were doing other changes at same time in which case you can silently drop it (not even worth a summary mention).  On that note, can 10 compat be added?

Reply:
>  On that note, can 10 compat be added?

havent tested (im not on 3.10 yet)

Reply:
Well, it's not urgent yet. I do suggest to add 3_10 alongside 3_9 to PYTHON_TARGETS if going to be working on some python ebuilds, would be nice to check if tests & runtime are fine for next version.